src/components/FreelancerCard.astro

```diff
@@ -78,21 +88,34 @@ const freelancerImage: ImageMetadata | undefined = imageFiles[imagePath];
         {`${freelancer.firstName} ${freelancer.lastName}`}
       </h2>
     </a>    <p class="text-pink-500 dark:text-pink-400 mb-2">{freelancer.headline}</p>
-    <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2">
-      📍 {freelancer.location} {getFlagEmoji(freelancer.countryCode)}
+    <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2 flex items-center gap-2">
+      📍 {freelancer.location} 
+      {flagImage && (
+        <Image 
+          src={flagImage}
+          alt={`Flag of ${freelancer.location}`}
+          width={20}
+          height={15}
+          class="inline-block"
+        />
+      )}
     </p>
     <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2">🗣️ {freelancer.language}</p>
     <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-6">Availability: {freelancer.availability}</p>
     <div class="flex items-center mb-4 mt-4 gap-2">
       <CQIcon profileUrl={freelancer.forumProfile} size="sm" />
       {freelancer.linkedIn && (
         <a href={freelancer.linkedIn} target="_blank" rel="noopener noreferrer" class="text-neutral-800 dark:text-neutral-100 inline-flex items-center">
-          <Icon name="mdi:linkedin" class="w-6 h-6 mr-2" /> 
+          <svg class="w-6 h-6 mr-2" viewBox="0 0 24 24">
+            <path fill="currentColor" d="M19 3A2 2 0 0 1 21 5V19A2 2 0 0 1 19 21H5A2 2 0 0 1 3 19V5A2 2 0 0 1 5 3H19M18.5 18.5V13.2A3.26 3.26 0 0 0 15.24 9.94C14.39 9.94 13.4 10.46 12.92 11.24V10.13H10.13V18.5H12.92V13.57C12.92 12.8 13.54 12.17 14.31 12.17A1.4 1.4 0 0 1 15.71 13.57V18.5H18.5M6.88 8.56A1.68 1.68 0 0 0 8.56 6.88C8.56 5.95 7.81 5.19 6.88 5.19A1.69 1.69 0 0 0 5.19 6.88C5.19 7.81 5.95 8.56 6.88 8.56M8.27 18.5V10.13H5.5V18.5H8.27Z" />
+          </svg>
         </a>
       )}
       {freelancer.github && (
         <a href={freelancer.github} target="_blank" rel="noopener noreferrer" class="text-neutral-800 dark:text-neutral-100 inline-flex items-center">
-          <Icon name="mdi:github" class="w-6 h-6 mr-2" /> 
+          <svg class="w-6 h-6 mr-2" viewBox="0 0 24 24">
```

Comment:
same as the comment above reg. the path inside the file vs. a simple readable line